lib/entry/FlavorDownloader.js

```diff
@@ -364,7 +365,35 @@ var FlavorDownloader = (function() {
         });
     }
 
-    FlavorDownloader.prototype.start = function (onIterationEndCallback) {
+    function getTotalBitrate(fallbackBitrate) {
+        if (this.encoderInformation && this.encoderInformation.Encoder) {
+            return this.encoderInformation.videoBitrate + this.encoderInformation.audioBitrate;
```

Comment:
what is there is no audio?  (or no video) in stream?  (for example audio only stream)